com.unity.render-pipelines.universal/Shaders/Nature/SpeedTree8Passes.hlsl

```diff
@@ -295,7 +296,7 @@ void InitializeInputData(SpeedTreeFragmentInput input, half3 normalTS, out Input
 
     inputData.fogCoord = input.interpolated.fogFactorAndVertexLight.x;
     inputData.vertexLighting = input.interpolated.fogFactorAndVertexLight.yzw;
-    inputData.bakedGI = half3(0, 0, 0); // No GI currently.
+    inputData.bakedGI = SAMPLE_GI(input.lightmapUV, input.interpolated.vertexSH, inputData.normalWS);
```

Comment:
This lines is giving errors. I assume it's because it needs to be `SAMPLE_GI(input.interpolated. interpolated.....)`

Reply:
Good catch, looks like the test setup did not actually turn on the GI, so the code wasn't even getting compiled.

Reply:
did you add test coverage for this code path?

Reply:
Still waiting for a reply...